app/main.py

```diff
@@ -1,44 +1,2 @@
-from fastapi import FastAPI, HTTPException
-from starlette.responses import Response
```

Comment:
Storing AWS secret keys or any sensitive information directly in the source code is a severe security risk. This can lead to unauthorized access and potentially compromise your AWS account and associated data. It's crucial to use environment variables or a secure secrets management service to handle such sensitive information.

**Label**: security

**Code suggestion**

```python
import os
aws_secret_key = os.getenv('AWS_SECRET_KEY')
```